### PR TITLE
feat(tests-unit): adds context/confirmations and context/services/notifications tests

### PR DESCRIPTION
**Pull request by jurajpiar**
Part of the https://rsklabs.atlassian.net/browse/RMKT-457 ticket.

Please review only the last two commits in this PR (previous come from https://github.com/rsksmart/rif-marketplace-ui/pull/727).

### CODE REVIEW

**Review by 0xartem — APPROVED** [review]

File: src/context/Confirmations/__tests__/actions.test.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  ConfirmationsRecord, ContractAction, NewConfirmationPayload, NewRequestPayload, State,
+} from '../interfaces'
+import actions from '../actions'
+
+const MOCK_THASH = 'MOCK_HASH'
+let state: State
+
+// juraj - TODO: test individual action-event relationships
+
+describe('Confirmations Context Actions', () => {
```
Comment (jurajpiar) [inline src/context/Confirmations/__tests__/actions.test.tsx:12]:
```suggestion
let state: State

describe('Confirmations Context Actions', () => {
```

**Review by itofarina — DISMISSED** [review]
Looks like this PR is rebased on #727 
Please, rebase this one against master once #727 is merged so reviewing it would be easier 🙂

**Comment by jurajpiar** [comment]
> Looks like this PR is rebased on #727
> Please, rebase this one against master once #727 is merged so reviewing it would be easier 🙂

Yes, as mentioned in the description @farinalvaro

**Review by itofarina — APPROVED** [review]
LGTM

**Pull request merged**